<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEditResult">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>计算选项</string>
    </property>
    <widget class="QCheckBox" name="checkBoxMD5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>16</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MD5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSHA1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA-1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSHA224">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>15</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA-224</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSHA256">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA-256</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSHA384">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA-384</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSHA512">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA-512</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCRC32">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>15</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CRC32</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>269</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>进度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>完成：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCheckCode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入待比对的校验码：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonComparison">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>302</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>比对</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBrowse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCopy">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>复制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
